docs/maintainer/infrastructure.md

```diff
@@ -469,6 +469,159 @@ uploaded from the new feedstock, by design.
+
+On issues:
+
+- `@conda-forge-admin, please...` command issues, handled by `@conda-forge-linter`.
+
+### Package building {#package-building}
+
+The pipelines that build conda packages are used for both pull requests and push events in `main` and other branches.
+The only difference is that the packages built during a pull request are not uploaded to the staging channel.
+Maintaining these up-to-date across all feedstocks involves several repositories:
+
+- `conda-smithy` is in charge of generating the CI pipelines themselves, together with the supporting scripts and configuration files.
+  These pipelines and scripts can rely on code and data defined in the repositories below.
+- `conda-forge-ci-setup-feedstock` provides the code needed to prepare and homogenize the CI runners across providers.
+  It also does some checks before the artifacts are uploaded to `cf-staging`.
+- `conda-forge-pinning-feedstock` defines which versions are supported for a number of runtimes and libraries, as well as the compilers used for certain languages and platforms.
+- `docker-images` builds the standardized container images for Linux runners.
+  This repository has additional authentication needs for Docker Hub, Quay.io.
+
+The pipelines can run on several CI providers supported by `conda-smithy`, including:
+
+- Azure DevOps Pipelines
+- Travis CI
+- Circle CI
+- Appveyor
+
+Registration of hooks and triggers is also done by the `conda-smithy` app.
+
+:::tip
+`conda-smithy` supports more CI providers.
+Check [its repository](https://github.com/conda-forge/conda-smithy) for more details.
+:::
+
+Authenticated services involved:
+
+- Anaconda.org uploads to `cf-staging`
+
+### Package validation and publication
+
+Once built on `main` (or other branches), the conda packages are uploaded to an intermediary channel named `cf-staging`.
+From there, the packages are downloaded by the validation server and, if successful, copied over to `conda-forge` itself.
+
+- The validation logic is defined at `conda-forge/artifact-validation`
+- If problematic, the results of the validation are posted as issues in the same repo.
+- This logic runs at `conda-forge/conda-forge-webservices`.
+  This web app also copies the artifacts from `cf-staging` to `conda-forge`.
+- Part of the validation includes checking for cross-package clobbering.
+  The list of authorized feedstocks per package name is maintained at `conda-forge/feedstock-outputs`.
+- Some further analysis might be performed _after_ publication.
+
+Authenticated services involved:
+
+- Anaconda.org uploads to `conda-forge`
+- The `conda-forge-webservices` app deployment itself (currently at Heroku)
+- (?) Post new issues to `conda-forge/artifact-validation`
+
+### Post-publication
+
+Once uploaded to anaconda.org/conda-forge, packages are not immediately available to CLI clients.
+They have to be replicated in the Content Distribution Network (CDN).
+This step usually takes less than 15 minutes.
```

Comment:
```suggestion
This step usually should ideally take around 15 minutes. In some circumstances, longer delays are possible. Check [conda-forge.org/status](https://conda-forge.org/status) in case of doubt.
```